pkgs/by-name/yo/yosys/package.nix

```diff
@@ -105,28 +110,45 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   enableParallelBuilding = true;
-  nativeBuildInputs = [
-    pkg-config
-    bison
-    flex
-  ];
-  propagatedBuildInputs = [
-    tcl
-    readline
-    libffi
-    zlib
-    (python3.withPackages (
-      pp: with pp; [
-        click
-      ]
-    ))
-  ] ++ lib.optional enablePython boost_python;
+  nativeBuildInputs =
+    [
+      bison
+      flex
+      pkg-config
+    ]
+    ++ lib.optionals enablePython [
+      (python3.withPackages (
+        p: with p; [
+          pip
+          setuptools
+          # wheel
+        ]
+      ))
+    ];
+
+  propagatedBuildInputs =
+    [
+      libffi
+      readline
+      tcl
+      zlib
+    ]
+    ++ lib.optionals enablePython [
+      (python3.withPackages (
+        pp: with pp; [
+          click
+        ]
+      ))
+      (boost.override {
+        enablePython = true;
+        python = python3;
+      })
```

Comment:
`python3.pkgs.boost` should also work here.

Reply:
Done.